wireshark/CMakeLists.txt

```diff
@@ -17,67 +17,46 @@
 # under the License.
 #
 
-set(CMAKE_CXX_FLAGS "-O3 -g ${CMAKE_CXX_FLAGS}")
+cmake_minimum_required(VERSION 3.7)
+project(pulsar-cpp-wireshark)
 
-MESSAGE(STATUS "Use WIRESHARK_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
+set(CMAKE_CXX_STANDARD 11)
 
```

Comment:
Addressed.

Reply:
The command should move in this line. Otherwise, it will use `#include <config.h>` by default, so when using a high version of Wireshark dependency will fail when built. 

```
➜  wireshark git:(bewaremypower/wireshark-fix) ✗ cmake --build build                         
[ 25%] Running cpp protocol buffer compiler on /Users/wangkai/Developer/github/pulsar-client-cpp/wireshark/../proto/PulsarApi.proto
[ 50%] Building CXX object CMakeFiles/pulsar-dissector.dir/pulsarDissector.cc.o
/Users/wangkai/Developer/github/pulsar-client-cpp/wireshark/pulsarDissector.cc:24:10: fatal error: 'config.h' file not found
#include <config.h>
         ^~~~~~~~~~
1 error generated.
gmake[2]: *** [CMakeFiles/pulsar-dissector.dir/build.make:84: CMakeFiles/pulsar-dissector.dir/pulsarDissector.cc.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/pulsar-dissector.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2

```

Reply:
I made this variable an extra search path in `find_path` (see https://cmake.org/cmake/help/latest/command/find_path.html). Does it work for you now?

Reply:
Yes, now it works.